smriprep/workflows/anatomical.py

```diff
@@ -620,3 +631,55 @@ def _pop(inlist):
     if isinstance(inlist, (list, tuple)):
         return inlist[0]
     return inlist
+
+
+def _aseg_two_three():
```

Comment:
A link to the reference used to build this would be a helpful comment.

Reply:
There's no link to this. I just scanned the freesurfer LUT and created a mapping myself (I had done something similar for the carpetplot). I tried to find some existing (prescribed with freesurfer tools) to do this task but found nothing. However, there are some labels for FSL-FAST segmentations in the freesurfer's LUT file - that makes me think there must be something.

I have looked at the generated segmentations and they look great - but some particular labels can be arguably changed (e.g., there are two lateral vessels that probably should be mapped to zero or to CSF, but because they are defined in a GM cluster of labels, I mistakenly included them).

You also found a naming problem to this function. My intention was:
```suggestion
def _aseg_to_three():
```

Reply:
What about https://github.com/freesurfer/freesurfer/blob/release_6_0_0/distribution/ASegStatsLUT.txt?